docs/userguide.md

```diff
@@ -239,6 +239,10 @@ select count(*) from account;
 1 row selected (1.97 seconds)
 ```
 
+## TiSparkR
+A thin layer build for supporting R language with TiSpark
+Please refer to [this document](../TiSparkR/README.md) for usage.
```

Comment:
The "please" here is overdoing the politeness. Generally do not use "please" in user guide. Reference: [https://developers.google.cn/style/tone#politeness](url)

Reply:
both done.